src/components/MetricsPane/MetricsPane.jsx

```diff
@@ -48,11 +50,10 @@ import { getIsSiteSelected, zoomToSelectedSite } from '../../helperFunctions/sel
 import { useMap } from 'react-map-gl'
 import { MAIN_MAP_ID } from '../../constants/constants'
 
-const MetricsPane = ({ showMetricsPane, setShowMetricsPane, view }) => {
+const MetricsPane = ({ showMetricsPane, setShowMetricsPane, view, showLoadingIndicator }) => {
   const [numSurveys, setNumSurveys] = useState(0)
   const [numTransects, setNumTransects] = useState(0)
   const [numUniqueCountries, setNumUniqueCountries] = useState(0)
-  const [yearRange, setYearRange] = useState('')
```

Comment:
We add some stuff to state that is already in reactive state. Since I worked with year, I cleaned it up a little. The rest of the values in `calculateMetrics` dont need extra reactive state, but I kept that out of scope. Its an optimization for a future dev.